app/src/test/java/com/ucmobiledevelopment/freeways/IncidentTests.kt

```diff
@@ -40,20 +40,20 @@ class IncidentTests {
 
     @Test
     fun `Given incident API is available When I fetch incidents from 2019 to 2020 in state 40 county 1 Then results should contain a case with caseId 400434`() = runTest{
-        GivenIncidentServiceIsInitialized()
-        WhenIncidentDataAreReadAndParsed()
-        ThenTheIncidentCollectionShouldContainIncidentWithCaseId400434()
+        givenIncidentServiceIsInitialized()
+        whenIncidentDataAreReadAndParsed()
+        thenTheIncidentCollectionShouldContainIncidentWithCaseId400434()
     }
 
-    private fun GivenIncidentServiceIsInitialized() {
+    private fun givenIncidentServiceIsInitialized() {
         incidentService = IncidentService()
     }
 
-    private suspend fun WhenIncidentDataAreReadAndParsed() {
+    private suspend fun whenIncidentDataAreReadAndParsed() {
         allIncidents = incidentService.fetchIncidents(2019, 2020, 40, 1)
     }
 
-    private fun ThenTheIncidentCollectionShouldContainIncidentWithCaseId400434() {
+    private fun thenTheIncidentCollectionShouldContainIncidentWithCaseId400434() {
```

Comment:
Renamed functions and renamed function calls.

Reply:
Good.